scripts/testlocal

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+echo "Moving your .env file to .env.testtemp"
+mv .env .env.testtemp
```

Comment:
might it be better to copy than move just in case this fails and you're left wondering where your env file went?

Reply:
I'm going to take it as resolved based on the thumbs up in slack. We can revisit if I am wrong on that count!